src/Domain/Regulation/Repository/RegulationOrderRecordRepositoryInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Regulation\Repository;
+
+use App\Domain\Regulation\RegulationOrderRecord;
+
+interface RegulationOrderRecordRepositoryInterface
+{
+    public function save(RegulationOrderRecord $regulationOrderRecord): RegulationOrderRecord;
+
+    public function findOneByUuid(string $uuid): RegulationOrderRecord|null;
+
+    public function findRegulations(): array;
```

Comment:
(Détail) En cohérence avec `findOneByUuid()` qui n'est pas `findOneRegulationByUuid()`, est-ce qu'on appellerait pas cette méthode comme ceci... ? Ce qui lèverait une ambiguité (il aurait fallu écrire `findRegulationOrderRecords()`, pour être rigoureux)

```suggestion
    public function findAll(): array;
```